### ISSUE TITLE
8.8 System.ArgumentOutOfRangeException: Length couldn't be less than zero.

### ISSUE THREAD

**Issue by Eugenii10** [d2phap/ImageGlass#1530]
### System information:
 - Windows OS version: Windows 11 version 21H2
 - ImageGlass version: 8.8.3.28 x64 Portable
 - Latest 4.8 .NET Framework is installed.

### To Reproduce
1. Launch ImageGlass.
2. Press 'X' to launch ExifTool.
3. Use next settings 'Tools > Command argumets':
    `-p "D:\Repos\exiftool-sd-format\exif.fmt"`
    The specified format file you can get here: https://raw.githubusercontent.com/Eugenii10/exiftool-sd-format/main/exif.fmt
4. If I don't use specified format file, it works as expected without any error. Previous version 8.7.11.6 worked fine with that format file.

### Actual behavior:
Additional popup window for ExifTool information is opened, but is is empty. The error message is displayed:
```
System.ArgumentOutOfRangeException: Длина не может быть меньше нуля.
Имя параметра: length
   в System.String.Substring(Int32 startIndex, Int32 length)
   в ImageGlass.Library.Image.ExifToolWrapper.ParseExifTags(String cmdOutput, String originalFileName)
   в ImageGlass.Library.Image.ExifToolWrapper.<ReadAsync>d__8.MoveNext()
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в ImageGlass.FrmExifTool.<Local_OnImageChanged>d__12.MoveNext()
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
```
It seems to be a problem with displaying tags from format file which are not presented in the current file's metadata. This is my guess.

### Expected behavior:
Opened popup window is filled with the information from ExifTool.

### Additional context:
Suddenly version 8.7.11.6 doesn't want to show popup window for ExifTool information. It just blinks once with it's title bar and nothing more happen: nor error, nor something visual happen. It even doesn't blink second time until you close it and reopen. Not sure what actions lead to that condition. And Windows restart doesn't help in this situation.

**Comment by d2phap** [comment]
Since v8.8, the default commands to pass to the exiftool has 1 more arugment: `-H` - which is to print out the TagId.
https://github.com/d2phap/ImageGlass/blob/0c5891f6f63e2235f9da021846c266f01b1f71c7/Source/Components/ImageGlass.Library/Image/ExifToolWrapper.cs#L47

But the UI currently only shows Tag group, Tag name and Tag value. The correct order should be:
```
Tag_Group\tTag_Id\tTag_Name\tTag_Value"
```

To make it works, you can change your fmt file as below:
```diff
# ***
- File	File name	$FileName
+ File	-	File name	$FileName
- File	Directory name	${directory;s/\//\\/g}
+ File	-	Directory name	${directory;s/\//\\/g}
```

Also note that, there is a bug that makes the preview command in both Settings > Tool and the Exiftool does not show `-H`.

**Comment by Eugenii10** [comment]
It works, thanks! I have just added `-` as Tag_Id to each line as you wrote.

**Comment by d2phap** [comment]
> Also note that, there is a bug that makes the preview command in both Settings>Tool and the Exiftool does not show `-H`.

It should be fixed in [v8.8.4.4](https://github.com/d2phap/ImageGlass/releases/tag/8.8.4.4).
